webapp/packages/core-ui/src/Tabs/Tab/Tab.tsx

```diff
@@ -54,7 +66,7 @@ export const Tab = observer<TabProps>(function Tab({ tabId, title, menuContext,
             disabled={disabled}
             onClick={tab.handleOpen}
           >
-            <tab-container>{children}</tab-container>
+            <tab-container className={сontainerClassName}>{children}</tab-container>
```

Comment:
Please don't use props for specific inner elements, we will rewrite it to css modules with SContext in the future. Also currently please avoid to change anything related to tabs (we will migrate them all together later)

Reply:
Okay I will revert that part